js/test/wallet/createTransaction.test.ts

```diff
@@ -0,0 +1,194 @@
+import { clusterApiUrl, Connection, Keypair, LAMPORTS_PER_SOL, PublicKey, SystemProgram } from '@solana/web3.js';
+import BigNumber from 'bignumber.js';
+import { createTransaction } from '../../src/wallet/createTransaction';
+
+describe('createTransaction', () => {
+    let connection: Connection;
+    let wallet: Keypair;
+
+    beforeAll(async () => {
+        const endpoint = clusterApiUrl('devnet');
+        connection = new Connection(endpoint, 'confirmed');
+
+        wallet = Keypair.generate();
+
+        const airdropSignature = await connection.requestAirdrop(wallet.publicKey, LAMPORTS_PER_SOL / 100);
+        await connection.confirmTransaction(airdropSignature);
+    });
+
+    describe('when transferring native SOL', () => {
+        describe('transaction', () => {
+            it('creates a transaction without memo', async () => {
+                expect.assertions(1);
+
+                const transaction = await createTransaction(
+                    connection,
+                    wallet.publicKey,
+                    new PublicKey('mvines9iiHiQTysrwkJjGf2gb9Ex9jXJX8ns3qwf2kN'),
+                    new BigNumber(0.01),
+                    {}
+                );
+
+                expect(transaction.instructions).toHaveLength(1);
+                // FIXME: Best way to validate?
```

Comment:
@jordansexton unsure on the best way to validate the correct transaction was created

Reply:
One quick way to do it would be to create the expected transaction and serialize it, and then compare the serialized bytes / base58 encoding.